<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BScanSegmentation</class>
 <widget class="QWidget" name="BScanSegmentation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="seriesTab">
      <attribute name="title">
       <string>Series</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>From segmentation</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Segmentation</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox"/>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="buttonSeriesInitFromSeg">
            <property name="text">
             <string>Create</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>From Threshold</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="0">
           <widget class="QRadioButton" name="radioButton_2">
            <property name="text">
             <string>Relative</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QRadioButton" name="radioButton_3">
            <property name="text">
             <string>From above</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Strikes</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="radioButton">
            <property name="text">
             <string>Absolute</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="spinBox"/>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="spinBox_2"/>
          </item>
          <item row="5" column="1">
           <widget class="QPushButton" name="buttonSeriesInitFromThreshold">
            <property name="text">
             <string>Create</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="radioButton_4">
            <property name="text">
             <string>From below</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>137</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="bscanTab">
      <attribute name="title">
       <string>BScan</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>From Threshold</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="3" column="0">
           <widget class="QRadioButton" name="radioButton_5">
            <property name="text">
             <string>Relative</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QRadioButton" name="radioButton_6">
            <property name="text">
             <string>From above</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_2"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Strikes</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="radioButton_7">
            <property name="text">
             <string>Absolute</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="spinBox_3"/>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="spinBox_4"/>
          </item>
          <item row="5" column="1">
           <widget class="QPushButton" name="buttonBScanInitFromThreshold">
            <property name="text">
             <string>Create</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="radioButton_8">
            <property name="text">
             <string>From below</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>GroupBox</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QToolButton" name="buttonBScanDilate">
            <property name="text">
             <string>Dilate</string>
            </property>
            <property name="icon">
             <iconset resource="../application.qrc">
              <normaloff>:/icons/arrow_out.png</normaloff>:/icons/arrow_out.png</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QToolButton" name="buttonBScanErode">
            <property name="text">
             <string>Erode</string>
            </property>
            <property name="icon">
             <iconset resource="../application.qrc">
              <normaloff>:/icons/arrow_in.png</normaloff>:/icons/arrow_in.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QToolButton" name="buttonBScanOpenClose">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../application.qrc">
              <normaloff>:/icons/arrow_inout.png</normaloff>:/icons/arrow_inout.png</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QToolButton" name="buttonBScanMedian">
            <property name="text">
             <string>Median</string>
            </property>
            <property name="icon">
             <iconset resource="../application.qrc">
              <normaloff>:/icons/arrow_inout.png</normaloff>:/icons/arrow_inout.png</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="localTab">
      <attribute name="title">
       <string>Local</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="2" column="2">
        <layout class="QGridLayout" name="gridLayout_6">
         <property name="margin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="2">
          <widget class="QToolButton" name="buttonLocalThresholdUp">
           <property name="text">
            <string>from below</string>
           </property>
           <property name="icon">
            <iconset resource="../application.qrc">
             <normaloff>:/icons/arrow_up.png</normaloff>:/icons/arrow_up.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QToolButton" name="buttonLocalThresholdRight">
           <property name="text">
            <string>from left</string>
           </property>
           <property name="icon">
            <iconset resource="../application.qrc">
             <normaloff>:/icons/arrow_right.png</normaloff>:/icons/arrow_right.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QToolButton" name="buttonLocalThresholdLeft">
           <property name="text">
            <string>from right</string>
           </property>
           <property name="icon">
            <iconset resource="../application.qrc">
             <normaloff>:/icons/arrow_left.png</normaloff>:/icons/arrow_left.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QToolButton" name="buttonLocalThresholdDown">
           <property name="text">
            <string>from above</string>
           </property>
           <property name="icon">
            <iconset resource="../application.qrc">
             <normaloff>:/icons/arrow_down.png</normaloff>:/icons/arrow_down.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="4">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="6" column="1">
        <widget class="QRadioButton" name="localThresholdRelative">
         <property name="text">
          <string>Relative</string>
         </property>
        </widget>
       </item>
       <item row="17" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="9" column="2">
        <widget class="QSpinBox" name="localThresholdStrikes"/>
       </item>
       <item row="6" column="2">
        <widget class="QDoubleSpinBox" name="localThresholdRelativeSpinBox"/>
       </item>
       <item row="10" column="0" colspan="3">
        <widget class="QRadioButton" name="radioLocalPaint">
         <property name="text">
          <string>Paint</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1" colspan="2">
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QGridLayout" name="gridLayout_7">
          <property name="margin">
           <number>0</number>
          </property>
          <item row="1" column="2">
           <widget class="QToolButton" name="toolButton_15">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QToolButton" name="localPaintRec">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../application.qrc">
              <normaloff>:/icons/paint_quadrat.png</normaloff>:/icons/paint_quadrat.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QToolButton" name="toolButton_13">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QToolButton" name="localPaintCircle">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../application.qrc">
              <normaloff>:/icons/paint_disc.png</normaloff>:/icons/paint_disc.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QToolButton" name="toolButton_14">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="14" column="1" rowspan="2" colspan="2">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="toolButton_10">
            <property name="text">
             <string>Erode</string>
            </property>
            <property name="icon">
             <iconset resource="../application.qrc">
              <normaloff>:/icons/arrow_in.png</normaloff>:/icons/arrow_in.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_9">
            <property name="text">
             <string>Dilate</string>
            </property>
            <property name="icon">
             <iconset resource="../application.qrc">
              <normaloff>:/icons/arrow_out.png</normaloff>:/icons/arrow_out.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QRadioButton" name="radioLocalThreshold">
         <property name="text">
          <string>Threshold</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="1">
        <widget class="QRadioButton" name="localThresholdAbsolut">
         <property name="text">
          <string>Absolute</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QSpinBox" name="localThresholdAboluteSpinbox"/>
       </item>
       <item row="8" column="1" rowspan="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Strikes</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0" colspan="3">
        <widget class="QRadioButton" name="radioLocalOperation">
         <property name="text">
          <string>Operation</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Direction</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Size</string>
         </property>
        </widget>
       </item>
       <item row="11" column="2">
        <widget class="QSpinBox" name="spinBox_5">
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../application.qrc"/>
 </resources>
 <connections/>
</ui>
